apps/events/migrations/0011_modify_schedulemessage_class.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.2.11 on 2024-06-19 17:15
+from django_celery_beat.models import PeriodicTask
+from django.db import migrations
+
+from apps.events.forms import EventActionForm
+from apps.events.models import EventActionType, ScheduledMessage
```

Comment:
You might change this as I see it is a WIP, but when migrating models you need to do use `apps.get_model` instead of accessing the actual model: https://docs.djangoproject.com/en/5.0/topics/migrations/#data-migrations